### PR TITLE
Update add-or-remove-members-from-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note to the document to stress the fact that only global admin or group admins can make changes to a group via the 365 admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2376

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/create-groups/add-or-remove-members-from-groups.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ In Microsoft 365, group members typically create their own groups, add themselve
 
 ::: moniker-end
 
+> [!NOTE]
+> You need to be a Global admin or a Groups admin to make changes to a group in the M365 admin center.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/add-or-remove-members-from-groups.md]:
```suggestion
> You need to be a global admin or a groups admin to make changes to a group in the Microsoft 365 admin center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@msbemba the required permissions for the procedures in this article are already listed in the paragraph prior to the procedure.

**Pull request closed without merging**